### PR TITLE
Tooltips on Maven artifact/group IDs

### PR DESCRIPTION
**Pull request by elharo**
@chanseokoh I blatantly stole these from the Cloud Dataflow plugin.

### CODE REVIEW

File: plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/Messages.java [hunk 1]
```diff
@@ -35,4 +36,12 @@ public static String getString(String key) {
       return '!' + key + '!';
     }
   }
+  
+  public static String getString(String key, Object... params  ) {
```
Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/Messages.java]: Extra space after `params`.

File: plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/Messages.java [hunk 2]
```diff
@@ -35,4 +36,12 @@ public static String getString(String key) {
...
+    try {
+      return MessageFormat.format(RESOURCE_BUNDLE.getString(key), params);
+    } catch (MissingResourceException e) {
```
Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/Messages.java]: `e` --> `ex`

File: plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/MavenAppEngineStandardWizardPage.java [hunk 3]
```diff
@@ -302,11 +306,11 @@ static boolean validateLocation(String location, WizardPage page) {
         FilePermissions.verifyDirectoryCreatable(path);
         return true;
       } catch (FileAlreadyExistsException ex) {
-          String message = MessageFormat.format(Messages.getString("FILE_LOCATION"), location); //$NON-NLS-1$
+          String message = Messages.getString("FILE_LOCATION", location); //$NON-NLS-1$
           page.setMessage(message, ERROR);
           return false;  
       } catch (IOException ex) {
-        String message = MessageFormat.format(Messages.getString("INVALID_PATH"), location); //$NON-NLS-1$
+        String message = Messages.getString("INVALID_PATH", location); //$NON-NLS-1$
```
Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/MavenAppEngineStandardWizardPage.java]: There are other usages of `MessageFormat` in this class. If you want, you can remove them (and the import).

File: plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/Messages.java (hunk 2 above)
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/Messages.java]: done

**Comment by chanseokoh** [comment]
LGTM

**Pull request merged**